pkg/pf/tests/testdata/TestDetailedDiffList/attribute_requires_replace/added.golden

```diff
@@ -25,7 +25,6 @@ Plan: 1 to add, 0 to change, 1 to destroy.
     +-testprovider:index/test:Test: (replace)
         [id=test-id]
         [urn=urn:pulumi:test::project::testprovider:index/test:Test::p]
-      ~ id  : "test-id" => output<string>
```

Comment:
This is wrong, will fix in https://github.com/pulumi/pulumi-terraform-bridge/issues/2660

Similar for all other cases when an ID diff is not displayed for a replace.